fe/fe-core/src/main/java/com/starrocks/analysis/UpdateBackendAddressClause.java

```diff
@@ -0,0 +1,45 @@
+package com.starrocks.analysis;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.starrocks.common.AnalysisException;
+import com.starrocks.common.Pair;
+import com.starrocks.system.SystemInfoService;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+
+public class UpdateBackendAddressClause extends BackendClause {
+
+    protected Pair<String, Integer> newhostPortPair;
```

Comment:
What's the new and what's old?
The name has a non-sense meaning.
It's better to use a more concise name.